### ISSUE TITLE
Cannot update/install to ESP chip

### ISSUE THREAD

**Issue by tamimology** [hydazz/docker-esphome#2]
This happened after the latest update a few days back. this is the log I see when I try to Update/clean install on my existing ESP

```
INFO Reading configuration /config/tv-back-leds.yaml...
INFO Generating C++ source...
INFO Compiling app...
Processing tv-back-leds (board: esp01_1m; framework: arduino; platform: platformio/espressif8266 @ 3.2.0)
--------------------------------------------------------------------------------
HARDWARE: ESP8266 80MHz, 80KB RAM, 1MB Flash
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
PermissionError: [Errno 13] Permission denied: '/piolibs/ArduinoJson':
  File "/usr/lib/python3.10/site-packages/platformio/builder/main.py", line 184:
    env.SConscript("$BUILD_SCRIPT")
  File "/config/.esphome/platformio/packages/tool-scons/scons-local-4.3.0/SCons/Script/SConscript.py", line 597:
    return _SConscript(self.fs, *files, **subst_kw)
  File "/config/.esphome/platformio/packages/tool-scons/scons-local-4.3.0/SCons/Script/SConscript.py", line 285:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "/config/.esphome/platformio/platforms/espressif8266/builder/main.py", line 238:
    target_elf = env.BuildProgram()
  File "/config/.esphome/platformio/packages/tool-scons/scons-local-4.3.0/SCons/Util.py", line 742:
    return self.method(*nargs, **kwargs)
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/platformio.py", line 64:
    env.ProcessProjectDeps()
  File "/config/.esphome/platformio/packages/tool-scons/scons-local-4.3.0/SCons/Util.py", line 742:
    return self.method(*nargs, **kwargs)
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/platformio.py", line 145:
    project_lib_builder = env.ConfigureProjectLibBuilder()
  File "/config/.esphome/platformio/packages/tool-scons/scons-local-4.3.0/SCons/Util.py", line 742:
    return self.method(*nargs, **kwargs)
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 1150:
    project.install_dependencies()
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 943:
    if _is_builtin(spec):
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 935:
    for lb in self.env.GetLibBuilders():
  File "/config/.esphome/platformio/packages/tool-scons/scons-local-4.3.0/SCons/Util.py", line 742:
    return self.method(*nargs, **kwargs)
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 1067:
    lb = LibBuilderFactory.new(env, lib_dir)
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 63:
    obj = globals()[clsname](env, path, verbose=verbose)
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 153:
    self.process_extra_options()
  File "/usr/lib/python3.10/site-packages/platformio/builder/tools/piolib.py", line 290:
    with fs.cd(self.path):
  File "/usr/lib/python3.10/site-packages/platformio/fs.py", line 37:
    os.chdir(self.new_path)
========================= [FAILED] Took 11.92 seconds =========================
```

**Comment by hydazz** [comment]
I am aware of this... open a shell to the container and run `chown -R abc:abc /piolibs`. this is done on startup however something keeps resetting the permissions to root. I will see why. above solution works until the container is recreated

**Comment by hydazz** [comment]
fixed d87d9fe 😑
